CRM/Core/Form.php

```diff
@@ -2576,17 +2576,40 @@ public function getCurrency($submittedValues = []) {
    * @return bool
    */
   protected function isFormInViewOrEditMode() {
+    return $form->isFormInViewMode() || $form->isFormInEditMode();
```

Comment:
Should be `$this`. Sorry my sample code was out of context.

Reply:
yikes, my bad!